tests/playwright/Report.md

```diff
@@ -0,0 +1,186 @@
+# Evaluation of Microsoft Playwright
+
```

Comment:
Create a "contents" section with links to all headings. Example: https://www.mediawiki.org/wiki/Selenium/Getting_Started/Run_tests_targeting_Beta_cluster